IABot/API.php

```diff
@@ -773,7 +773,12 @@ public static function getAllArticles( $limit, $resume ) {
 	* @copyright Copyright (c) 2016, Maximilian Doerr
 	* @return mixed Revid if successful, else false
 	*/
-	public static function edit( $page, $text, $summary, $minor = false, $timestamp = false, $bot = true, $section = false, $title = "", &$error = null ) {
+	public static function edit( $page, $text, $summary, $minor = false, $timestamp = false, $bot = true, $section = false, $title = "", &
+		$error = null ) {
+		if( TESTMODE ) {
+			echo $text;
```

Comment:
This doesn't seem like a good idea, since my output is the CLI itself.  It would probably be better have it print it to a file instead and name the file after the page name.